src/Helmet.js

```diff
@@ -273,6 +295,10 @@ const generateTagsAsString = (type, tags) => {
                 }
 
                 const encodedValue = encodeSpecialCharacters(tag[attribute]);
+                // itemProp in react is camelcase, but will generate as lowercase
```

Comment:
API should be html key `itemprop` as you would expect to see it in the DOM.  It will just pass-through in this function.